polyply/data/gromos_53A6_polymers/Hter.ff

```diff
@@ -0,0 +1,7 @@
+[ moleculetype ]
+; Name   nrexcl
+Hter    3
+;
+[ atoms ]
+;  nr  type  resnr  resid  atom  cgnr  charge    mass    total_charge
+    1    HC    1    P3HT   HTe    1    0.245    1.008 ;  0.245
```

Comment:
The resname cannot be P3HT because it is not the same residue. You could name it TER or HTer or whatever you want. I think with Martini my convention was calling it 'OHter' i.e. with small ter. But whatever floats your boat.

Reply:
Right! Fixed (https://github.com/marrink-lab/polyply_1.0/pull/112/commits/88bd340b9e0896e2da9f021e8586f5d18f25f7d9), I went for Hter.